main/modes/picross/mode_picross.c

```diff
@@ -92,8 +92,9 @@ void picrossStartGame(display_t* disp, font_t* mmFont, picrossLevelDef_t* select
     p->input->blinkError = false;
     p->input->blinkAnimTimer =0;
     p->input->startHeldType = OUTOFBOUNDS;
-    p->input->inputBoxColor = c430;
-    p->input->inputBoxDefaultColor = c430;
+    p->input->inputBoxColor = c005;//now greenish for more pop against marked. old burnt orange: c430. old green: c043
+        //lets test this game against various forms of colorblindness. I'm concerned about deuteranopia. Input square is my largest concern. 
```

Comment:
That's me! (but protanopia)

Reply:
Let me know what you think, readability wise! In general the input square needs to be thicker, but with 'guides' on it works well. So I turned them on by default. (also the comment is wrong, its blue-ish now)